Examples/BinaryBH/Main_BinaryBH.cpp

```diff
@@ -57,6 +59,13 @@ int runGRChombo(int argc, char *argv[])
 
     std::chrono::time_point<Clock> start_time = Clock::now();
 
+    // Add a scheduler to GRAMR to calls specificPostTimeStep on every AMRLevel
+    // at time 0
+    CallFunctionOverLevelsPtr<> call_now(
```

Comment:
I'm not sure about this name... :-) Maybe something like "multilevel_task" is more descriptive?